.github/actions/setup-rust/action.yml

```diff
@@ -50,7 +50,5 @@ runs:
 
     - name: Install ${{ inputs.install }}
       if: ${{ inputs.install != '' }}
-      uses: actions-rs/cargo@v1
-      with:
-        command: install
-        args: ${{ inputs.install }}
+      shell: bash
```

Comment:
Why here? And is this needed in other places?

Reply:
It was only complaining here, I think it's because it's an action and not a workflow